server/embed/config.go

```diff
@@ -163,7 +170,14 @@ type Config struct {
 	SnapshotCatchUpEntries uint64 `json:"experimental-snapshot-catch-up-entries"`
 
 	MaxSnapFiles uint `json:"max-snapshots"`
-	MaxWalFiles  uint `json:"max-wals"`
+	// MaxWalFiles is the maximum number of write ahead log files stored in the wal directory.
```

Comment:
minor issue: add empty line before introducing `Deprecated`. For example, https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/io/ioutil/ioutil.go;l=87

Reply:
Done. Thanks for the heads up. In previous PRs, I deprecated without adding that empty line. I added a task under #17578 to track this.